webapp/src/pages/EditAvatarPage/EditAvatarPage.tsx

```diff
@@ -70,35 +63,28 @@ export default function EditAvatarPage() {
         },
       })
       .then(() => {
-        setStatus({
-          type: 'success',
-          message: 'Image visible area saved correctly.',
-        });
+        notify('Image visible area saved correctly.');
```

Comment:
And now we only need to call a function hook, no need to manage notification states on each component

Reply:
praise: Thanks for updating these pages!